molecule/default/molecule.yml

```diff
@@ -31,6 +31,8 @@ platforms:
   #   image: ubuntu:xenial
   # - name: ubuntu18
   #   image: ubuntu:bionic
+  # - name: ubuntu20
```

Comment:
Should we update the list of commented out platforms in `meta/main.yml` to correspond to all the commented out platforms listed here for testing? Once this has migrated to Python 3 I imagine we will want to turn on testing for all those platforms.

Reply:
I think we can just add them to `meta/main.yml` when the time comes.  I was just following what was already present in the repo.

Reply:
Ok, only reason I asked is other repos have them included and commented out.